### ISSUE TITLE
Make Polar preserve scroll position and zoom level after opening/closing a document

### ISSUE THREAD

**Issue by emilga** [burtonator/polar-bookshelf#317]
Currently, both scroll position and zoom level is reset after updating to a newer version of Polar.

And zoom level is reset after closing/reopening a document.

**Comment by likyng** [comment]
perhaps this should be two issues

**Comment by burtonator** [comment]
Yes.. we also have to fix issues with zoom with HTML documents.. but I agree both position and zoom should be preserved...
